spec/fabricators/grpc/history_event_fabricator.rb

```diff
@@ -130,7 +125,7 @@ class TestSerializer
   event_type { Temporal::Api::Enums::V1::EventType::EVENT_TYPE_ACTIVITY_TASK_CANCELED }
   activity_task_canceled_event_attributes do |attrs|
     Temporal::Api::History::V1::ActivityTaskCanceledEventAttributes.new(
-      details: TestSerializer.to_details_payloads('ACTIVITY_ID_NOT_STARTED'),
+      details: Temporal::Configuration.new.converter.to_details_payloads('ACTIVITY_ID_NOT_STARTED'),
```

Comment:
Shouldn't this be `$converter.to_details_payloads(...)`? Not sure why you are allocating a new one here.

Reply:
You're right, forgot to replace this one 👍